core/modules/qdisp/TaskSpec.h

```diff
@@ -1,63 +0,0 @@
-// -*- LSST-C++ -*-
```

Comment:
It's used for people who use emacs. There is an LSST-DM emacs c++ mode that is pre-customized to LSST coding standards. I (as an emacs user) would prefer that it not be removed. The lsst emacs modes can be found at: https://dev.lsstcorp.org/cgit/LSST/DMS/devenv/build.git/